.kokoro/tests/run_tests.sh

```diff
@@ -55,6 +55,7 @@ if [[ "$SCRIPT_DEBUG" != "true" ]]; then
 
     # Setup required env variables
     export GOOGLE_CLOUD_PROJECT=java-docs-samples-testing
+    export GOOGLE_CLOUD_PROJECT_NUMBER="779844219229" # For Transcoder samples
```

Comment:
Let's change this to TRANSCODER_PROJECT_NUMBER

Reply:
Done